src/forms/editors/MultipleChoiceEditor.jsx

```diff
@@ -97,24 +100,35 @@ export default class MultipleChoiceEditor extends Component {
         <div style={ styles.options }>
           {
             this.state.options.map((option, i) => {
-                return (
-                  <div key={ i } style={ styles.optionRow }>
-                    <div style={ styles.optionRowText }>
-                      <input style={ styles.optionInput } type="text" value={ option.title } onChange={ this.updateOption.bind(this, i) } />
-                    </div>
-                    <div style={ styles.optionRowButtons }>
-                      <button style={ styles.optionButton } onClick={ this.duplicateOption.bind(this, i) }><FaCopy /></button>
-                      {
-                        (i > 0) || (i == 0 && this.state.options.length > 1) ?
-                          <button style={ styles.optionButton } onClick={ this.removeOption.bind(this, i) }><FaTrashO /></button>
-                        :
-                          <button style={ styles.optionButton } disabled><FaTrashO /></button>
+              return (
+                <div key={ i } style={ styles.optionRow }>
+                  <div style={ styles.optionRowText }>
+                    <input
+                      ref={
+                        (el) => {
```

Comment:
this section should either be pulled out to a named method or given some comments. I think i know what's going on here, but only because I'm looking for it.